.claude/skills/adversarial-reviewer/SKILL.md

```
@@ -0,0 +1,184 @@
+
+## Anti-Patterns (Immediate Flags)
+
+- "It works locally" assumptions
+- Implicit type coercion in critical paths
+- Business logic in UI layer
+- Silent fallbacks
+- Catch + ignore
+
+## Output Format (Strict)
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)